ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayAnnotation/README.metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "category": "TODO",
```

Comment:
Looks like these TODOs need filled in.

Reply:
Yep, the metadata.json file is what powers the samples on the [developer site](https://developers.arcgis.com/qt/latest/qml/sample-code/analyze-hotspots/). It is a lot of duplicate info from the readme